src/loaders/compute_tools/gtdb_tk/gtdb_tk.py

```diff
@@ -74,6 +80,12 @@ def _run_gtdb_tk(
     print(f'Start executing GTDB-TK for {size} genomes')
     start = time.time()
 
+    mash_db_path = output_dir / MASH_DB_PATH
+    if mash_db_path.exists():
+        # in the event of a rerun, remove an existing Mash database since we want GTDB-TK to create a new one
```

Comment:
```suggestion
        # in the event of a rerun, remove an existing Mash database since we want GTDB-TK to create a new one
        # as the inputs might be a different set of assembly files
```